### ISSUE TITLE
Never Ending Related Anime Data Downloading

### ISSUE THREAD

**Issue by ElementalCrisis** [ShokoAnime/ShokoServer#705]
## VERSION INFORMATION

Server Version: 3.8.3
Desktop Version: 3.8.3

## LOG FILE

[2018-02-03.txt](https://github.com/ShokoAnime/ShokoServer/files/1695451/2018-02-03.txt)

## DESCRIPTION

I created a test DB and added a single show, [009-1](https://anidb.net/perl-bin/animedb.pl?show=anime&aid=4637) and Shoko went on a non-stop related anime downloading spree jumping from series to series. I attached a portion of the log as reference. 

## STEPS TO REPRODUCE

Not sure if this will work bur worth a shot.

1.  Create new DB.
2. Add series 009-01

**Comment by ElementalCrisis** [comment]
OK I think I figured it out. 

I believe its because of Seiyuus, this series https://anidb.net/perl-bin/animedb.pl?show=anime&aid=13266 is not in my collection nor is any show that is related to it. However, I do have a seres in my collection that this man (https://anidb.net/perl-bin/animedb.pl?show=creator&creatorid=37) did a voice for and he is one of the seiyuu's listed in the series I linked above. 

So basically, I'm getting a ton of series data and images for series I'll probably never watch let alone add to my collection.

**Comment by da3dsoul** [comment]
Rawr it's not like I'm excessively familiar with that section of the code base....

**Comment by ElementalCrisis** [comment]
Still an issue.

[2018-04-07.txt](https://github.com/ShokoAnime/ShokoServer/files/1886254/2018-04-07.txt)

**Comment by Cazzar** [comment]
3db847f0ceb82f446011c1dae853d7925d6f4a1b and 24a367e199168f0170e061af7d932a31213b939b should resolve this issue.
Relation downloading will limit itself to a depth of 3 (configurable)
